distribution metrics for cloud run and azure container apps (#16920)
DataDog/documentation: content/en/serverless/azure_container_apps/_index.md

{{< /programming-lang-wrapper >}}

### Custom metrics
You can submit custom metrics using a [DogStatsD client][4].

> For monitoring Cloud Run and other serverless applications, use [distribution][7] metrics.

Reviewer: ```suggestion
For monitoring Azure App Service and other serverless applications, use [distribution][8] metrics.
```

> Distributions provide `avg`, `sum`, `max`, `min`, and `count` aggregations by default. On the Metric Summary page, you can enable percentile aggregations (p50, p75, p90, p95, p99) and also manage tags. To monitor a distribution for a gauge metric type, use `avg` for both the [time and space aggregations][8]. To monitor a distribution for a count metric type, use `sum` for both the time and space aggregations.

Reviewer: ```suggestion
Distributions provide `avg`, `sum`, `max`, `min`, and `count` aggregations by default. On the Metric Summary page, you can enable percentile aggregations (p50, p75, p90, p95, p99) and also manage tags. To monitor a distribution for a gauge metric type, use `avg` for both the [time and space aggregations][9]. To monitor a distribution for a count metric type, use `sum` for both the time and space aggregations.
```

> For more information, see [Serverless Custom Metrics - Understanding distribution metrics][6].

Reviewer: ```suggestion
For more information, see [Serverless Custom Metrics - Understanding distribution metrics][7].
```

[4]: /metrics/custom_metrics/dogstatsd_metrics_submission/
[5]: /getting_started/site/
[6]: /getting_started/tagging/unified_service_tagging/

> [7]: /serverless/custom_metrics/#understanding-distribution-metrics

Reviewer 2: The rest of the content of that page is about Lambda, so I'm worried that linking user from a azure container apps doc to a lambda doc could cause confusion. I think we can probably leave out the link or we should refactor that section from the lambda doc to a standalone guide.
